src/sql/src/plan/statement/ddl.rs

```diff
@@ -3482,13 +3482,16 @@ pub fn plan_drop_item(
 
                 let dep = scx.catalog.get_item(id);
                 if dependency_prevents_drop(object_type, dep) {
+                    // TODO: Add a hint to add cascade.
                     sql_bail!(
                         "cannot drop {}: still depended upon by catalog item '{}'",
                         scx.catalog.resolve_full_name(catalog_entry.name()),
                         scx.catalog.resolve_full_name(dep.name())
                     );
                 }
             }
+            // TODO(jkosh44) It would be nice to also check if any active subscribe relies on
```

Comment:
One way to solve this would be to copy the Global Ids of all subscribe dependencies into `scx`. I just wanted to get some opinions on whether or not that would be wort it before implementing it.

Reply:
What happens in this case? Does the SUBSCRIBE stall forever?

Reply:
What happens with this PR is that, even if `CASCADE` is not specified, we allow people to drop an object that a SUBSCRIBE depends on and we stop the SUBSCRIBE. To answer the question directly, no the SUBSCRIBE does not stall forever.

Ideally (I think) we'd like to prevent users from dropping an object that a SUBSCRIBE depends on, unless they specify `CASCADE`.